src/subdomains/core/aml/enums/aml-error.enum.ts

```diff
@@ -36,4 +38,48 @@ export enum AmlError {
   SUSPICIOUS_MAIL = 'SuspiciousMail',
 }
 
-export const AmlPendingError = [AmlError.IP_MISMATCH, AmlError.SUSPICIOUS_MAIL, AmlError.WEEKLY_LIMIT_REACHED];
+export const AmlPendingError = [
+  AmlError.IP_MISMATCH,
+  AmlError.SUSPICIOUS_MAIL,
+  AmlError.WEEKLY_LIMIT_REACHED,
+  AmlError.NAME_CHECK_WITHOUT_KYC,
+  AmlError.NAME_CHECK_WITH_BIRTHDAY,
+];
```

Comment:
You can remove this and just check, if the map entry below is not null for this error.